src/iks007_nonlinear2Dsolid.cpp

```diff
@@ -55,7 +55,7 @@ int main(int argc, char **argv) {
   //  //  /// ALUGrid Example
   //  using Grid = Dune::ALUGrid<gridDim, 2, Dune::simplex, Dune::conforming>;
   //  auto grid  =
-  //  Dune::GmshReader<Grid>::read("testFiles/unstructuredTrianglesfine.msh",
+  //  Dune::GmshReader<Grid>::read("auxiliaryFiles/unstructuredTrianglesfine.msh",
```

Comment:
We should fix the issue where the different grids are not working.

We should also uncomment the code and add some #define at the top where the people can switche between the examples like https://godbolt.org/z/ff1seedKv . But maybe the preprocessor can also use strings and not numbers then it would be more self explainatory. We can look into it.

Reply:
This will be done in a seperate PR. #1